Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -44,17 +45,29 @@ typedef NS_ENUM(NSInteger, OCTMessageFileType) {
 
 typedef NS_ENUM(NSInteger, OCTMessageCallEvent) {
     /**
-     * Call dialing
+     * Call was answered.
      */
-    OCTMessageCallEventDial,
+    OCTMessageCallEventAnswered,
 
     /**
-     * Call was missed.
+     * Call was unanswered.
      */
-    OCTMessageCallEventMissed,
+    OCTMessageCallEventUnanswered,
+};
+
+typedef NS_ENUM(NSInteger, OCTCallStatus) {
+    /**
+     * Call is currently ringing.
+     */
+    OCTCallStatusRinging,
+
+    /**
+     * Call is currently dialing a chat.
+     */
+    OCTCallStatusDialing,
 
     /**
-     * Call has ended.
+     * Call is currently active in session.
      */
-    OCTMessageCallEventEnd,
+    OCTCallStatusActive,
```

Comment:
Don't worry you weren't the only one :smile: . In reality it should just be `MuteAll`. However I will just borrow the naming convention from toxav for now. Maybe we can change it later down the road to avoid any confusion